repose-aggregator/tests/functional-tests/build.gradle

```diff
@@ -24,6 +24,7 @@ dependencies {
     integrationTestCompile "org.slf4j:jcl-over-slf4j"
     integrationTestCompile "org.opensaml:opensaml-core"
     integrationTestCompile "org.opensaml:opensaml-saml-impl"
+    integrationTestCompile "xmlunit:xmlunit"
```

Comment:
I used it to verify that the response was not altered in the Flow 1.0 case for XML responses.